src/Articulate.Web/Web.config

```diff
@@ -36,7 +36,7 @@
       Umbraco web.config configuration documentation can be found here: 
       http://our.umbraco.org/documentation/using-umbraco/config-files/#webconfig
       -->
-    <add key="umbracoConfigurationStatus" value="" />
+    <add key="umbracoConfigurationStatus" value="7.5.6" />
```

Comment:
Changes to those files should have not been pushed. Umbraco version should be empty or a fresh clone will not trigger the installation of umbraco